src/core/public/doc_links/doc_links_service.ts

```diff
@@ -289,6 +289,7 @@ export class DocLinksService {
           migrationApiDeprecation: `${ELASTICSEARCH_DOCS}migration-api-deprecation.html`,
           nodeRoles: `${ELASTICSEARCH_DOCS}modules-node.html#node-roles`,
           releaseHighlights: `${ELASTICSEARCH_DOCS}release-highlights.html`,
+          version8ReleaseHighlights: `${ELASTIC_WEBSITE_URL}guide/en/elastic-stack/8.0/elastic-stack-highlights.html`,
```

Comment:
Q: Will we need to revisit the `8.0` bit in the URL in future minors?

Reply:
We'll have to look at it in future releases yes. If there is a reason to keep the `8.0` and create a new `8.1` we'll do that. For now, upgrade assistant that consumes this link is disabled in `8.0` and `8.1`.